src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+import { createAction } from '@reduxjs/toolkit/src/createAction';
+
+const INITIAL_STATE = [];
+
+const categoriesReducer = (state = INITIAL_STATE, actions) => {
+  if (actions.type === 'COMPLETED') {
+    return 'Under Construction';
+  } if (actions.type === 'IN_PROGRESS') {
+    return 'Under Construction';
+  }
```

Comment:
- [ ] Please make sure that you define a variable `CHECK_STATUS` as an action type.
  * The value of the action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`

- [ ]  `NOTE` : You have two action types here that are returning the same value so why don't you use only one?